styles/list.css

```diff
@@ -0,0 +1,31 @@
+.list_main__container{
+    display: flex;
+    flex-direction: column;
+    border: 1px solid var(--primary-color);
+}
+
+.normal-list li{
+    padding: 1rem;
+    margin-bottom: 1rem;
+    background-color: var(--grey);
+    border: hidden hidden 1px solid var(--primary-color) ;
+    text-align: center;
+}
```

Comment:
What is hidden hidden doing here? You might want to check this out.

Reply:
i actually want to show only bottom border line, but here it wont work here , i will fix it